src/main/webapp/WEB-INF/themes/desktop/xform/article-transform.xsl

```diff
@@ -1413,6 +1413,7 @@
               </xsl:attribute>
               <xsl:attribute name="alt">thumbnail</xsl:attribute>
               <xsl:attribute name="class">thumbnail</xsl:attribute>
+              <xsl:attribute name="loading">lazy</xsl:attribute>
```

Comment:
Looks like we could make this change for mobile too:
```
10:38 $ rg -g **/*.xsl img ../wombat
../wombat/src/main/webapp/WEB-INF/themes/root/xform/jpub3-html.xsl
2063:    <img alt="{@xlink:href}">
2070:    </img>

../wombat/src/main/webapp/WEB-INF/themes/mobile/xform/article-transform.xsl
1243:          <img alt="thumbnail" class="figure-image">
1247:          </img>
1375:    <xsl:element name="img">

../wombat/src/main/webapp/WEB-INF/themes/desktop/xform/article-transform.xsl
1395:        <div class="img-box">
1409:            <xsl:element name="img">
1618:    <xsl:element name="img">

```

Reply:
safari on mac/ios doesn't support it ([yet](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Browser_compatibility)) and its the lion's share of our mobile traffic.
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/5441006/82360156-cf0fa880-99bd-11ea-8e46-6af39dcd3d42.png">
But can't hurt either :)